packages/yoastseo/spec/languageProcessing/helpers/sentence/getSentencesFromTreeSpec.js

```diff
@@ -11,7 +11,7 @@ describe( "test to get sentences from the tree", () => {
 	} );
 	it( "returns the sentences from paragraph and heading nodes", () => {
 		const paper = new Paper( "<div><p>A very intelligent cat loves their human. A dog is very cute.</p><h3>A subheading 3" +
-			"</h3>text text text<h4>A subheading 4</h4>more text.</div>" );
+			"</h3><h4>A subheading 4</h4></div>" );
```

Comment:
We removed the implicit paragraphs from this test since they aren't a realistic use case, and instead created a separate test to test implicit paragraphs below.